css/src/atomics/flex.scss

```diff
@@ -57,3 +57,28 @@ $flex-atomics: (
 		}
 	}
 }
+
+@include tablet {
+	@each $key in map.keys($flex-atomics) {
+		@each $value in map.get($flex-atomics, $key) {
+			.#{$key}-#{$value}-tablet {
+				#{$key}: $value !important;
+			}
+		}
+	}
+}
+
+@if $include-flex-desktop == true {
```

Comment:
I think this is a cool strategy we talked about, but for now, perhaps we should just remove this. We can circle back later (😆 if/when -desktop) becomes necessary.